src/index.html

```diff
@@ -19,9 +19,423 @@
+              <img
+                src="images/imac.jpeg"
+                alt="APPLE A1419 iMac 27"
+                class="card__img"
+              />
+            </div>
+            <div class="card__name">
+              <h3 class="card__title">
+                APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+              </h3>
+              <p class="card__product-code">Product code: 195434</p>
+            </div>
+            <div class="card__review-block">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <p class="card__review-text">Reviews: 5</p>
+            </div>
+            <div class="card__price-block">
+              <p class="card__price-text">Price:</p>
+              <p class="card__price">$2,199</p>
+            </div>
+            <div class="card__button button">
+              <a
+                href="#"
+                class="button__link"
+                data-qa="card-hover"
+              >
+                BUY
+              </a>
+            </div>
+          </div>
+        </div>
+        <div class="products__card card">
+          <div class="card__container">
+            <div class="card__image">
+              <img
+                src="images/imac.jpeg"
+                alt="APPLE A1419 iMac 27"
+                class="card__img"
+              />
+            </div>
+            <div class="card__name">
+              <h3 class="card__title">
+                APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+              </h3>
+              <p class="card__product-code">Product code: 195434</p>
+            </div>
+            <div class="card__review-block">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <p class="card__review-text">Reviews: 5</p>
+            </div>
+            <div class="card__price-block">
+              <p class="card__price-text">Price:</p>
+              <p class="card__price">$2,199</p>
+            </div>
+            <div class="card__button button">
+              <a
+                href="#"
+                class="button__link"
+                data-qa="card-hover"
+              >
+                BUY
+              </a>
+            </div>
+          </div>
+        </div>
+        <div class="products__card card">
+          <div class="card__container">
+            <div class="card__image">
+              <img
+                src="images/imac.jpeg"
+                alt="APPLE A1419 iMac 27"
```

Comment:
This is a repetition of the same block of code for the product card. As mentioned earlier, consider using a templating engine or a JS framework to dynamically generate these cards.